#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_396895_HOTPLUG_conf_man.dpatch by Y Giridhar Appaji Nag <giridhar@appaji.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Explain HOTPLUG_INTERFACES in the ifplugd.conf(5) manpage.

@DPATCH@
diff -urNad ifplugd-0.28~/man/ifplugd.conf.5 ifplugd-0.28/man/ifplugd.conf.5
--- ifplugd-0.28~/man/ifplugd.conf.5	2008-04-20 12:47:01.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.conf.5	2008-04-20 12:48:24.000000000 +0530
@@ -11,6 +11,9 @@
 \fBINTERFACES\f1
 Specifies the ethernet interfaces to monitor. It has to contain a space seperated list of network interfaces names. Most users will probably use "eth0" here, however you may add additional interfaces for monitoring more than one device. A special value is supported as well: "auto" will enable a more or less working auto detection of available network devices. This won't make you happy when using network module auto loading, since it cannot detect currently unloaded network devices.
 .TP
+\fBHOTPLUG_INTERFACES\f1
+Specifies the interfaces that can be hotplugged (like interfaces on PCMCIA, USB or WLAN adapters).  "all" can be used to make the udev script start an ifplugd process for any hotplugged interfaces (except those already listed in INTERFACES).
+.TP
 \fBARGS\f1
 Additional command line arguments for ifplugd invocation. See \fBifplugd(8)\f1 for further information.
 .TP
diff -urNad ifplugd-0.28~/man/ifplugd.conf.5.xml.in ifplugd-0.28/man/ifplugd.conf.5.xml.in
--- ifplugd-0.28~/man/ifplugd.conf.5.xml.in	2008-04-20 12:47:01.000000000 +0530
+++ ifplugd-0.28/man/ifplugd.conf.5.xml.in	2008-04-20 12:47:56.000000000 +0530
@@ -51,12 +51,20 @@
 	  </option>
 
 	  <option>
+		<p><opt>HOTPLUG_INTERFACES</opt></p> <optdesc><p>Specifies the
+		interfaces that can be hotplugged (like interfaces on PCMCIA,
+		USB or WLAN adapters).  "all" can be used to make the udev
+		script start an ifplugd process for any hotplugged interfaces
+		(except those already listed in INTERFACES).
+		</p></optdesc>
+	  </option>
+
+	  <option>
 		<p><opt>ARGS</opt></p> <optdesc><p>Additional command line
 		arguments for ifplugd invocation. See <manref name="ifplugd"
 		section="8"/> for further information.</p></optdesc>
 	  </option>
 
-
 	  <option>
 	       <p><opt>ARGS_</opt><arg>iface</arg></p>
                   
